README.md

```diff
@@ -138,6 +159,7 @@ To run the samples please refer to the corresponding tutorials:
   - [Image Classification sample](examples/tensorflow/classification/README.md)
   - [Object Detection sample](examples/tensorflow/object_detection/README.md)
   - [Instance Segmentation sample](examples/tensorflow/segmentation/README.md)
+- [ONNX sample](examples/post_training_quantization/onnx/mobilenet_v2/README.md)
```

Comment:
Should we highlight somehow that this is a PTQ sample and not the same as the QAT samples above?

Reply:
done

Reply:
Can we resolve it?